### PR TITLE
docs(meetings): agenda meeting for July

### PR DESCRIPTION
**Pull request by lirantal**
Fixes https://github.com/nodejs/security-wg/issues/561

### CODE REVIEW

File: meetings/2019-07-15.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Node.js Foundation Security WorkGroup Meeting 2019-07-15
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: $GITHUB_ISSUE$
+* **Minutes Google Doc**: $MINUTES_DOC$
```
Comment (cjihrig) [inline meetings/2019-07-15.md]:
```suggestion
* **Minutes Google Doc**: https://docs.google.com/document/d/1Lx6QF7qQ4q_PIxVrE3-xpqKVGRXJ3kgfrdbL6o1fX4U/edit
```

File: meetings/2019-07-15.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
+# Node.js Foundation Security WorkGroup Meeting 2019-07-15
+
+## Links
+
+* **Recording**:  
+* **GitHub Issue**: $GITHUB_ISSUE$
```
Comment (cjihrig) [inline meetings/2019-07-15.md]:
```suggestion
* **GitHub Issue**: https://github.com/nodejs/security-wg/issues/561
```

File: meetings/2019-07-15.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# Node.js Foundation Security WorkGroup Meeting 2019-07-15
+
+## Links
+
+* **Recording**:
```
Comment (cjihrig) [inline meetings/2019-07-15.md]:
```suggestion
* **Recording**:  https://www.youtube.com/watch?v=ADJQRq2gvDo&feature=youtu.be
```

**Comment by lirantal** [comment]
thanks @cjihrig

**Pull request merged**